queries.sql

```diff
@@ -8,3 +8,80 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+----------------------------------------------
+
+-- update the animals table by setting the species column to unspecified
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
```

Comment:
- [ ] I can see there is no changes in your `Schema.sql` file. As requested in project requirement, let's modify your `schema.sql` file by adding a column `species` of type `string` to your `animals` table.